src/gridstack.ts

```diff
@@ -497,10 +497,12 @@ export class GridStack {
     let domNodes: GridStackNode[];
     if (column === 1 && this.opts.oneColumnModeDomSort) {
       domNodes = [];
-      this.$el.children('.' + this.opts.itemClass).each((index, el: GridItemHTMLElement) => {
-        let node = el.gridstackNode;
-        if (node) { domNodes.push(node); }
-      });
+      Array.from(this.el.children)

```

Comment:
thank you! I think there is a 4th one (checks for ':not(.' + this.opts.placeholderClass + ') which we can always do, and should be extracted into a convenience routine.
I can do that.